db/migrate/20220913201337_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
[Hint]: You can do it like that:
```ruby
add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
```

Reply:
> * [ ]  Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
>   [Hint]: You can do it like that:
> 
> ```ruby
> add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
> ```

- [ ] As requested by the previous reviouwer 🙃  make sure the reference is created on the `author_id` column and not user_ide Kindly addrees this!!